src/row.js

```diff
@@ -361,6 +361,7 @@ Row.prototype.createRules = function(rules, gaxOptions, callback) {
 
   var reqOpts = {
     tableName: this.table.id,
+    appProfileId: this.appProfileId,
```

Comment:
I think we will need to locate `appProfileId` from `this.bigtable.appProfileId`.